XML/TestCases/FunctionalTests-SGX.xml

```diff
@@ -0,0 +1,15 @@
+<TestCases>
+    <test>
+        <testName>VALIDATE-INTEL-SGX-DRIVER-FOR-DC-VM</testName>
+        <testScript>validate-intel-sgx-driver.sh</testScript>
+	<timeout>1800</timeout>
+        <setupType>OneVM</setupType>
+        <OverrideVMSize>Standard_DC2s</OverrideVMSize>
+        <files>.\Testscripts\Linux\validate-intel-sgx-driver.sh,.\Testscripts\Linux\utils.sh</files>
+        <Platform>Azure</Platform>
+        <Category>Functional</Category>
+        <Area>SGX</Area>
+        <Tags>sgx</Tags>
+        <Priority>0</Priority>
```

Comment:
Please be informed; the priority 0 means this test case runs all the times in the most of new release kernel. Is it the goal? @paulcallen  @Francis-Liu

Reply:
@harz566 @LiliDeng  @simonxiaoss  Agree?

Reply:
Call out this comment and take this 'yes'. Let's keep it as '0'